checkout-example-advanced/index.js

```diff
@@ -140,7 +140,7 @@ app.post("/api/submitAdditionalDetails", async (req, res) => {
 });
 
 // Handle all redirects from payment type
-app.all("/api/handleShopperRedirect", async (req, res) => {
+app.all("/handleShopperRedirect", async (req, res) => {
```

Comment:
check: I see that we're redirecting to `/handleShopperRedirect`, do we want to keep consistency in our sample apps by keeping the `/api/` in front (or vice versa: change future apps to not include the `/api`-part), the latter makes more sense to me

Reply:
It is not an API call, this is why I have renamed (we can correct the other samples as we go)